libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.directive.ts

```diff
@@ -35,7 +35,7 @@ export class SkyAgGridRowDeleteDirective
   implements AfterContentInit, OnDestroy
 {
   /**
-   * Specifies the ids of the data in the rows on which inline delete's should be shown.
+   * The ids of the data in the rows on which inline delete's should be shown.
```

Comment:
```suggestion
   * The IDs of the data in the rows where the inline delete appears.
```
A couple things here, neither of which are related to your changes. `ID` should be capitalized as a style thing, and the current version of the description is garbled. `on which inline delete's should be shown` doesn't make sense. I'm not sure when this text was added and I'm just taking my best guess as to what we mean here. I'm happy to take another look if my suggestion is inaccurate. But if you don't catch me online today, I wouldn't delay this PR to clean up this one item.

Reply:
These suggestions aren't related to the changes from your regular expression, but we need to fix the grammatically nonsensical wording we currently have. If you're good with making the change here, can you check if my suggested changes work? Or if you want me to handle these changes separately, let me know so that I can file a bug to track that work. Thanks!